app/gen-server/lib/homedb/UsersManager.ts

```diff
@@ -0,0 +1,759 @@
+import { ApiError } from "app/common/ApiError";
+import { normalizeEmail } from 'app/common/emails';
+import { UserOrgPrefs } from "app/common/Prefs";
+import * as roles from 'app/common/roles';
+import {
+  ANONYMOUS_USER_EMAIL,
+  EVERYONE_EMAIL,
+  FullUser,
+  PermissionDelta,
+  PREVIEWER_EMAIL,
+  UserOptions,
+  UserProfile
+} from "app/common/UserAPI";
+import { AclRule } from "app/gen-server/entity/AclRule";
+import { Group } from "app/gen-server/entity/Group";
+import { Login } from "app/gen-server/entity/Login";
+import { User } from "app/gen-server/entity/User";
+import { appSettings } from "app/server/lib/AppSettings";
+import { flatten } from "lodash";
+import { EntityManager } from "typeorm";
+import { HomeDBManager, PermissionDeltaAnalysis, Scope } from "../HomeDBManager";
```

Comment:
Maybe you can stay consistent with absolute path notation of the rest of the document rather than relative path notation.

Reply:
Done!